_posts/2020-04-16-remote-junior-developer-tips.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: 'Tips From One Remote Junior Developer to Another.'
+date: 2020-04-19 10:30:00
+categories: ["junior", "remote"]
+author: fionaDL
+---
+
+Becoming the Apprentice, and then Junior Developer at OmbuLabs was both exciting and nerve racking. Learning a new company can be challenging, especially when the skills are new and the work is remote.
```

Comment:
@FionaDL Maybe you can link to https://www.ombulabs.com/blog/tags/remote when you mention `remote`

```suggestion
Becoming the Apprentice, and then Junior Developer at OmbuLabs was both exciting and nerve racking. Learning a new company can be challenging, especially when the skills are new and the work is [remote](https://www.ombulabs.com/blog/tags/remote).
```